internal/app/config.go

```diff
@@ -18,6 +18,7 @@ type Config struct {
 	Dev          bool
 	DB           string `default:"mongodb://localhost"`
 	Auth0        Auth0Config
+	Mailer       MailerConfig
```

Comment:
```suggestion
	STMP         SMTPConfig
	SendGrid     SendGridConfig
```

Avoid deep config struct